### PR TITLE
fix:Loan account detail error layout

### LINKED ISSUE #314: Make image and textview at centre
**Summary:**

As you can see the in this GIF, there a image and textview that is on the top but I want those at the centre.

### PR DESCRIPTION
**Pull request by dilpreet2028**
Fixes #314 

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `MifosStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

File: app/src/main/res/layout/fragment_loan_account_details.xml [hunk 1]
```diff
@@ -1,16 +1,12 @@
...
-        xmlns:app="http://schemas.android.com/apk/res-auto"
-        android:orientation="vertical">
-
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    xmlns:tools="http://schemas.android.com/tools"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+    <ScrollView
+        android:layout_width="fill_parent"
```
Comment (therajanmaurya) [inline app/src/main/res/layout/fragment_loan_account_details.xml]: fill_parent is depreciated?

Comment (dilpreet2028) [reply app/src/main/res/layout/fragment_loan_account_details.xml]: Didn't noticed fill_parent, making changes

Comment (therajanmaurya) [reply app/src/main/res/layout/fragment_loan_account_details.xml]: During GSoC application, some buddy did this and It's accidentally merged my me. I think.
Thanks for PR.

**Comment by therajanmaurya** [comment]
@dilpreet96 in future use this layout for showing the error 
https://github.com/openMF/self-service-app/blob/development/app/src/main/res/layout/layout_error.xml

**Comment by dilpreet2028** [comment]
Okay @therajanmaurya, this layout is also using layout_error. I will check other layouts also and make sure layout_error is used.

**Pull request merged**